test/util/testConstructorConfig.json

```diff
@@ -68,5 +68,6 @@
   "publicFolder": "constructor",
   "favicon": "constructor",
   "versionedPublic": "constructor",
-  "alwaysHostPublic": "constructor"
+  "alwaysHostPublic": "constructor",
+  "cores": 1
```

Comment:
Set this to "constructor" like the other items. It looks silly, but all we're testing for here is that this param value is what gets set.